src/aws-cpp-sdk-core/source/utils/component-registry/ComponentRegistry.cpp

```diff
@@ -42,7 +42,6 @@ namespace Aws
             void ShutdownComponentRegistry()
             {
                 std::unique_lock<std::mutex> lock(s_registryMutex);
-                assert(s_registry);
```

Comment:
I would prefer to keep these asserts, but it will crash apps build in debug mode.